### ISSUE TITLE
Conflicting guidance regarding Service creation in "Scale Your App" tutorial

### ISSUE THREAD

**Issue by RBBIT** [kubernetes/website#46429]
I find this small bit in the scale-intro a bit confusing:

> NOTE If you are trying this after [the previous section ](https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/), then you may have deleted the service you created, or have created a Service of type: NodePort. In this section, it is assumed that a service with type: LoadBalancer is created for the kubernetes-bootcamp Deployment.
> 
> If you have not deleted the Service created in [the previous section](https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro), first delete that Service and then run the following command to create a new Service with its type set to LoadBalancer:
> 
> kubectl expose deployment/kubernetes-bootcamp --type="LoadBalancer" --port 8080

and then directly under that paragraph, there is this highlighted "Note" section, that states:

> Note:
> If you are trying this after [the previous section](https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/), you may have deleted the Service exposing the Deployment. In that case, please expose the Deployment again using the following command:
> 
> kubectl expose deployment/kubernetes-bootcamp --type="NodePort" --port 8080

TL;DR;
Paragraph instructs me to delete the "NodePort" service, because this sections expects a "Loadbalancer" service to be active. Highlighted Note instructs me to re-create the "NodePort" service after the "Loadbalancer" service was created.

**Comment by dipesh-rawat** [comment]
_Page reported in issue (based on the issue title):_  https://kubernetes.io/docs/tutorials/kubernetes-basics/scale/scale-intro/
/language en

**Comment by dipesh-rawat** [comment]
/retitle Conflicting guidance regarding Service creation in "Scale Your App" tutorial
/kind bug

**Comment by dipesh-rawat** [comment]
Removing this ambiguity will improve the tutorial's usability.

/priority backlog
/triage accepted
